systemservices/marketplace/src/events/index.ts

```diff
@@ -0,0 +1,72 @@
+import { hexToAscii, fromUnit, findInAbi, parseTimestamp } from "../contracts/utils";
+import { ABIDefinition } from "web3/eth/abi";
+import _marketplaceABI from "../contracts/Marketplace.abi.json"
+const marketplaceABI = _marketplaceABI as ABIDefinition[]
+
+const eventHandlers: {[ethName: string]: {
```

Comment:
handlers are usually function, this should be renamed to `eventDefinition`, `eventMapping`...

I also don't like much this way of managing the events, this creates a huge dependency between the events from the smart contract and the events from the service (that are different even if we are just doing a proxy). 

We could have a `eventMapping` that does the mapping between smart contract to the data needed for the events on the marketplace but we will still have separated events.

eg:
```js
eventMapping = {
  smartcontracteventX: event => ({ x: ... })
}

// event/serviceCreated.ts
export data => mesg.emitEvent('serviceCreated', eventMapping['smartcontracteventX'](data)
```